worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java

```diff
@@ -232,7 +233,7 @@ private int processHandles(List<Future<EobSearchResult>> futureHandles, Progress
     }
 
     private int writeOutResource(List<IBaseResource> eobs, StreamHelper helper, Versions.FhirVersions version) {
-        var jsonParser = fhirContext.newJsonParser();
+        var jsonParser = Versions.getContextFromVersion(version).newJsonParser();
```

Comment:
I thought this would be removed and instead pull a JSON parser from the already initialized ones.

Reply:
Whoops, fixed